tests/app/delivery/test_send_to_providers.py

```diff
@@ -626,7 +630,7 @@ def test_should_set_notification_billable_units_if_sending_to_provider_fails(
         send_to_providers.send_sms_to_provider(sample_notification)
 
     assert sample_notification.billable_units == 1
-    assert mock_toggle_provider.called
+    # assert mock_toggle_provider.called
```

Comment:
A better approach here might be to uncomment lines 624 and 633 and to assert on line 633 that dao_toggle_sms_provider is not called.

Reply:
I took my advice.